requirements.txt

```diff
@@ -1,6 +1,6 @@
-discord.py==1.3.4
+discord.py==2.2.2
 python-dotenv>=0.9.1
-pytz==2018.5
-requests==2.20.0
-beautifulsoup4==4.6.3
-psycopg2==2.8.4
+pytz==2022.7.1
+requests==2.28.2
+beautifulsoup4==4.12.0
+psycopg2-binary==2.9.5
```

Comment:
In my personal projects and local environment I always have to use -binary, and that was true here. I have never gotten the non-binary to work. Hopefully this isn't a problem, but I believe this is the recommended way anyway.